gnovm/pkg/gnolang/preprocess.go

```diff
@@ -3572,7 +3572,13 @@ func evalStaticType(store Store, last BlockNode, x Expr) Type {
 	pn := packageOf(last)
 	// See comment in evalStaticTypeOfRaw.
 	if store != nil && pn.PkgPath != uversePkgPath {
-		pv := pn.NewPackage() // temporary
+		pv := &PackageValue{
+			Block: &Block{
+				Source: pn,
+			},
+			PkgName: pn.PkgName,
+			PkgPath: pn.PkgPath,
+		}
```

Comment:
It seems the only logical change here is the exclusion of the realm-related logic?The performance improvement is definitely compelling, but a clear comment explaining the reasoning would be helpful. 

Reply:
The biggest difference is not for the realm-related logic, but for the usage of [PrepareNewValues](https://github.com/gnolang/gno/pull/4521#issue-3239517476). PrepareNewValues allocates a _lot_ of memory, and so avoiding it for the case of `evalStaticType`, which actually doesn't really refer to the values in the PackageValue, makes the whole ordeal much cheaper to process.